phpunit.xml.dist

```diff
@@ -67,5 +68,12 @@
         <var name="privileged_db_port" value="3306"/>
         -->
         <env name="COLUMNS" value="120"/>
+        <env name="DOCTRINE_DEPRECATIONS" value="trigger"/>
```

Comment:
I used `env` rather than `server` as described in https://github.com/doctrine/deprecations/?tab=readme-ov-file#displaying-deprecations-after-running-a-phpunit-test-suite for the sake of consistency.